client/state/document/test/reducer.js

```diff
@@ -0,0 +1,136 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+import deepFreeze from 'deep-freeze';
+
+/**
+ * Internal dependencies
+ */
+import {
+	DOCUMENT_LINK_ADD,
+	DOCUMENT_LINK_REMOVE,
+	DOCUMENT_META_ADD,
+	DOCUMENT_META_REMOVE,
+	DOCUMENT_DESCRIPTION_SET,
+	DOCUMENT_TITLE_SET,
+  DOCUMENT_UNREAD_COUNT_SET,
+} from 'state/action-types';
+
+import {
+	description,
+	link,
+	meta,
+	title,
+  unread_count,
+} from '../reducer';
+
+describe.only( 'reducer', () => {
+	describe( '#title()', () => {
+		it( 'should default to an empty string', () => {
+			const state = title( undefined, {} );
+
+			expect( state ).to.eql( '' );
+		} );
+
+		it( 'should properly set a new title', () => {
+			const newState = title( undefined, { type: DOCUMENT_TITLE_SET, title: 'new title'} );
+
+			expect( newState ).to.eql( 'new title' );
```

Comment:
I left a similar comment at https://github.com/Automattic/wp-calypso/pull/4172/files#r56832322, but this should probably be `equal`, not `eql`, unless you need deep comparison.